ApeWorX/ape
# fix: improve plugins install config schema

### What I did

fixes: #151 

### How I did it

* Created a helper method that extracts the module name and the package name from the config, following the new schema defined in the corresponding issue.

### How to verify it

* The following yaml should install a specific version of the solidity plugin and the latest version of the ledger plugin:

```yaml
plugins:
  - solidity:
      version: 0.1.0a2
  - ledger:

```

* A missing config, empty config, or empty `plugins:` in a config should not install anything nor should it error, it just should do nothing.

e.g. 

```yaml
plugins:

```

* A malformed plugin config entry item should raise an error and display the expected schema for installing plugins via the config.

### Checklist
